spaces/domains.json

```
@@ -478,4 +478,5 @@
   "vote.aavegotchi.com": "aavegotchi.eth",
   "snapshot.keplerdao.finance": "keplerdao.eth",
   "signal.paladin.vote": "palvote.eth"
+  "vote.alphadao.money": "alphagov.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "signal.paladin.vote": "palvote.eth",
  "vote.alphadao.money": "alphagov.eth"
```